actors/builtin/storage_miner/storage_miner_actor_state.go

```diff
@@ -211,13 +211,13 @@ func MinerInfo_New(
 		PendingWorkerKey: WorkerKeyChange{},
 	}
 
-	TODO() // TODO: determine how to generate/validate VRF key and initialize other fields
+	TODO() // TODO anorth: determine how to generate/validate VRF key and initialize other fields
```

Comment:
I don't know what this is talking about

Reply:
Same interaction set about how you envision the worker key relating to the workerID which I've not understood yet